src/Amadevus.RecordGenerator.Generators/BuilderPartialGenerator.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using System;
```

Comment:
This change is out-of-context. Please revert.

Reply:
see comment above